service/controller/v2/key/key.go

```diff
@@ -158,19 +158,19 @@ func WorkerSubnetName(customObject providerv1alpha1.AzureConfig) string {
 	return fmt.Sprintf("%s-%s-%s", ClusterID(customObject), virtualNetworkSuffix, workerSubnetSuffix)
 }
 
-func MasterVMSSName(customObject providerv1alpha1.AzureConfig) string {
-	return fmt.Sprintf("%s-master", ClusterID(customObject))
-}
-
-func WorkerVMSSName(customObject providerv1alpha1.AzureConfig) string {
-	return fmt.Sprintf("%s-worker", ClusterID(customObject))
+func MasterInstanceName(customObject providerv1alpha1.AzureConfig, instanceID string) string {
```

Comment:
Would be cool to align the godoc in this file. Half of the functions have it, the other half doesn't.
No blocker though <3